File: SocketsCS/Persistence/Interfaces/IRepoExcursie.cs
using System;
using System.Collections.Generic;
using Model;

namespace Persistence.Interfaces
{
    public interface IRepoExcursie: IRepo<Guid, Excursie>
    {
        IEnumerable<Excursie> CautaExcursii(string obiectivTuristic, int deLaOra, int panaLaOra);
    }
}

File: SocketsJava/Persistence/src/main/java/agentie/persistence/Interfaces/IRepoExcursie.java
package agentie.persistence.Interfaces;

import agentie.model.Excursie;

import java.util.UUID;

public interface IRepoExcursie extends IRepo<UUID, Excursie> {
    Iterable<Excursie> cautaExcursii(String numeObiectiv, Integer deLaOra, Integer panaLaOra);
}

